### PR TITLE
updated calling hero sample to latest stable release 1.22.0

### PR DESCRIPTION
**Pull request by alkwa-msft**
Updated to 1.22.0 and minor dialer improvements!

### CODE REVIEW

File: Calling/src/app/views/HomeScreen.tsx [hunk 1]
```diff
@@ -131,7 +131,8 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
 
   const isMobileSession = useIsMobile();
 
-  const showDisplayNameField = !teamsIdentityChosen;
+  let showDisplayNameField = true;
+  showDisplayNameField = !teamsIdentityChosen;
```
Comment (mgamis-msft) [inline Calling/src/app/views/HomeScreen.tsx:135]: We should revert this change for simplification

**Review by mgamis-msft — APPROVED** [review]

**Review by dmceachernmsft — APPROVED** [review]

File: Calling/src/app/views/HomeScreen.tsx (hunk 1 above)
Comment (alkwa-msft) [reply Calling/src/app/views/HomeScreen.tsx:135]: This should have been a direct port from the calling sample in UI Lib so we may need to also simplify this statement there as well

Comment (alkwa-msft) [reply Calling/src/app/views/HomeScreen.tsx]: image
